packages/react-network-diagrams/lib/components/Rack.js

```diff
@@ -58,9 +58,50 @@ var Rack = exports.Rack = function (_React$Component) {
                 fill: this.props.fill
             });
         }
+    }, {
+        key: "buildRmuArray",
+        value: function buildRmuArray(childElements, rmuCount, inchToRmu) {
+            //initialize an array of objects the size of the rack for the front and back
+
+            var frontRmuArray = Array(rmuCount + 1).fill({});
+            var rearRmuArray = Array(rmuCount + 1).fill({});
+
+            //Lets place the equipment at the RMU position on the front and on the back
+            //and fill the other spots up to its height
+            childElements.forEach(function (child) {
+                var childValues = child.props;
+                var rmuHeight = childValues.equipmentHeight / inchToRmu;
+                var start = Number(childValues.rmu);
```

Comment:
@srichmond the problem was here, I added `Number()` conversion, as `childValues.rmu` is a string, and therefore calculating `end` in the line below was wrong ("12"+1 became 121 instead of 13)